lighthouse-core/config/full-config.js

```diff
@@ -19,6 +20,7 @@ module.exports = {
   audits: [
     'byte-efficiency/unused-javascript',
   ],
+  // @ts-ignore TODO(bckenny): type extended Config where e.g. category.title isn't required
```

Comment:
If a config is extending another one, there's no reason for it to have *anything* in particular defined, so maybe these could be `RecursivePartial<LH.Config.Json>`, but we don't want to apply that to e.g. `defaultConfig`. So we'll have to figure out a good way to conditionally apply it another day.

Reply:
Maybe we have to split the full config typedef from the valid config typedef?

Reply:
> Maybe we have to split the full config typedef from the valid config typedef?

yeah, that seems like that might be the only decent way to do it without making the full config typedef useless